contracts/callback-capturer/NOTICE

```diff
@@ -0,0 +1,14 @@
+CW1-Whitelist: A minimal CosmWasm proxy contract
+Copyright (C) 2020 Confio OÜ
```

Comment:
Can you adjust this? Or ideally remove all licenses from the contract and only use the one from the repo.